lib/services/saml.go

```diff
@@ -687,10 +687,22 @@ func (o *SAMLConnectorV2) CheckAndSetDefaults() error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	if o.Metadata.Name == teleport.Local {
```

Comment:
This is a strangely specific check... So you expect it to be "local" sometimes, but how about "anybadvalue"?

Reply:
This is to make sure no one names their SAML (or OIDC) connector `local` because we now support `tsh --auth=local`.

Reply:
got it. I clearly remember that saw a better error message in your code for this purpose, you used to say that "local" is a reserved word and cannot be used to name a connector.

Reply:
Rolled back to the original message and original `CheckAndSetDefaults`.